test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -294,7 +294,7 @@ INSTANTIATE_TEST_CASE_P(
     mkIniTc("basic_dataset_p", {nw_adam, dataset, input, out+"input_layers=inputlayer"}, SUCCESS),
     mkIniTc("basic_dataset2_p", {nw_sgd, input, out+"input_layers=inputlayer", dataset}, SUCCESS),
     mkIniTc("basic_dataset3_p", {dataset, nw_sgd, input, out+"input_layers=inputlayer"}, SUCCESS),
-    mkIniTc("basic_conv2d_p", {nw_adam, conv2d + "input_shape = 1:1:62720"}, SUCCESS),
+    mkIniTc("basic_conv2d_p", {nw_adam, conv2d + "input_shape = 1:10:10"}, SUCCESS),
```

Comment:
i wonder how this unittest was even working till now

Reply:
Right. I am guessing that because we didn't have dimension check at `initialize`

Reply:
Then please add this unit test and let it fail.

Reply:
let me handle this later as well.